src/dymoprint/lib/utils.py

```diff
@@ -33,3 +31,40 @@ def draw_image(bitmap):
         yield drawobj
     finally:
         del drawobj
+
+
+def px_to_mm(px):
+    mm = px / PIXELS_PER_MM
+    # Round up to nearest 0.1mm
+    return math.ceil(mm * 10) / 10
+
+
+def is_debug_mode() -> bool:
+    def is_env_var_true(env_var: str) -> bool:
+        val = os.getenv(env_var)
+        return val is not None and val.lower() in ("1", "true")
+
+    return any(is_env_var_true(env_var) for env_var in ("DEBUG", "VERBOSE"))
+
+
+def print_exception(e):
+    if is_debug_mode():
+        LOG.exception(e)
+    else:
+        LOG.error(e)
+
+
+def configure_logging():
+    logging.basicConfig(
+        level=logging.DEBUG if is_debug_mode() else logging.INFO,
+        format="[%(levelname)s] %(message)s",
+    )
```

Comment:
The `logging.basicConfig` function is really barely suitable for only the smallest internal projects. The problem is that it operates on the root logger, so it has the potential to activate logging for libraries used by dymoprint. While it doesn't seem to cause any problems at the moment, it's much safer to operate only on the specific logger scoped to the `dymoprint` namespace.

```suggestion
    logger = logging.getLogger("dymoprint")
    logger.setLevel(logging.DEBUG if is_debug_mode() else logging.INFO)
    handler = logging.StreamHandler(sys.stderr)
    formatter = logging.Formatter("[%(levelname)s] %(message)s")
    handler.setFormatter(formatter)
    logger.addHandler(handler)
```